src/reanimated2/NativeReanimated.js

```diff
@@ -1,11 +1,13 @@
-import { TurboModuleRegistry, Platform } from 'react-native';
+import { Platform } from 'react-native';
 import reanimatedJS from './js-reanimated';
 
 const InnerNativeModule =
   Platform.OS === 'web'
     ? false
     : global.__reanimatedModuleProxy ||
-      TurboModuleRegistry.getEnforcing('NativeReanimated');
+      require('react-native').TurboModuleRegistry.getEnforcing(
```

Comment:
This still requires the `react-native` package to be in the package.json (not great for web-only projects). Perhaps this can be extracted out in a file with a `.native.js` and a `.web.js`?